api/services/Search/util.js

```diff
@@ -25,7 +25,7 @@ export const filterAndSortPosts = curry((opts, q) => {
   let { topics = [] } = opts
 
   const sortColumns = {
-    votes: 'posts.num_votes',
+    votes: 'posts.num_people_reacts',
```

Comment:
i feel like we should use this opportunity to change the sort name to reactions from votes